python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -235,10 +237,12 @@ def run_failure_sensor(
         description (Optional[str]): A human-readable description of the sensor.
         monitored_jobs (Optional[List[Union[JobDefinition, GraphDefinition, UnresolvedAssetJobDefinition, RepositorySelector, JobSelector]]]): The jobs
             in the current repository that will be monitored by this failure sensor. Defaults to None, which means the alert will
-            be sent when any job in the current repository fails. Jobs in external repositories can be specified using RepositorySelector and JobSelector.
+            be sent when any job in the current repository fails.
         job_selection (Optional[List[Union[JobDefinition, GraphDefinition, RepositorySelector, JobSelector]]]): (deprecated in favor of monitored_jobs)
             The jobs in the current repository that will be monitored by this failure sensor. Defaults to None, which means
             the alert will be sent when any job in the repository fails.
+        monitor_instance (bool): If set to True, the sensor will monitor all jobs in the Dagster instance. If used, you cannot also specify monitored_jobs
```

Comment:
A few nitpicks:
- Worth pointing out that this is mutually exclusive with `monitored_jobs`.
- It would make sense to put this right beneath `monitored_jobs`.
- Better to say "runs" than "jobs", because it will also monitor asset materializations that aren't part of jobs.
- Exceeds the 100-character line length limit.
- What do you think about calling this `monitor_all_repositories` to avoid the "instance" vs. "deployment" confusion.